data/json/recipes/recipe_ammo.json

```diff
@@ -117,11 +117,11 @@
   "reversible": false,
   "autolearn": true,
   "book_learn": [[ "glassblowing_book", 5 ]],
+  "qualities":[
+    {"id":"GRIP","level":3}
```

Comment:
Should I reformat all the crappy formatting or not? I can't seem to be able to do so without being criticized.